apps/site/src/app/apply/sections/Form/Form.tsx

```diff
@@ -46,7 +47,7 @@ export default function Form() {
 			const res = await axios.post(APPLY_PATH, formData);
 			if (res.status === 201) {
 				console.log("Application submitted");
-				// TODO: show submitted message
+				redirect("/portal");
```

Comment:
Couldn't get the redirect working, I think because this is a Client Component, so we need to call push with `useRouter`.